cmd/azure.go

```diff
@@ -103,7 +130,8 @@ func newAzureLayer(account, key string) (GatewayLayer, error) {
 		return AzureObjects{}, err
 	}
 	client := c.GetBlobService()
-	return &AzureObjects{client}, nil
+	meta := azureMultipart{meta: make(map[string]map[string]string)}
```

Comment:
You don't need a new variable. Two meta in same line is confusing. You can move this directly to the next line..